src/main/java/dev/slimevr/vr/trackers/TrackerRole.java

```diff
@@ -1,13 +1,11 @@
 package dev.slimevr.vr.trackers;
 
 /**
- * The SteamVR tracker role.
- *
  * The tracker role classifies the position and the role of a tracker on user's
- * body or playspace (like CAMERA or BEACON), using SteamVR naming. Tracker
- * roles are hints for interacting programs what the tracker means, and they do
- * not correspond to body's bones on purpose. Example: virtual vive trackers for
- * SteamVR vs actual SlimeVR trackers.
+ * body or playspace (like CAMERA or BEACON). Tracker roles are hints for
+ * interacting programs what the tracker means, and they do not correspond to
```

Comment:
I think we are accidentally overwriting the updated comment here, should be the one on the left

Reply:
That seem to be some github shenanigans showing diff, it's as it should be in the branch right now.